neurolib/optimize/evolution/evolution.py

```diff
@@ -912,6 +855,11 @@ def dfEvolution(self):
             for ip, p in enumerate(allIndividuals):
                 dfEvolution.loc[ip, f"f{i}"] = p.fitness.wvalues[i] / self.weightList[i]
 
+        # the history keeps all individuals of all generations
+        # there can be duplicates (in elitism for example), which we filter
+        # out for the dataframe
+        dfEvolution = self._dropDuplicatesFromDf(dfEvolution)
+        dfEvolution = dfEvolution.reset_index(drop=True)
```

Comment:
Bugfix: had to move `drop_duplicates()` to here, before it was happening to early and produced `nan`s